plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/organize/imports/ImportsFactory.xtend

```diff
@@ -1,90 +0,0 @@
-/**
```

Comment:
Why replacing Xtend implementation with Java?

Reply:
I was adding new case for the namespaces, which resulted in 3 new methods that were very similar to the already existing. With java it was easier to use method references and compose them, then with xtend.
Also, java has better hot swap, better editor, better content assist. In general, nowadays I favour Java over Xtend.